source/workflowapi/external_resources.json

```diff
@@ -115,24 +115,13 @@
                   "Action": [
                     "states:ListStateMachines",
                     "states:TagResource",
-                    "states:CreateStateMachine"
-                  ],
-                  "Resource": "*"
-                },
-                {
-                  "Effect": "Allow",
-                  "Action": [
+                    "states:CreateStateMachine",
```

Comment:
There seems to be a bug with stepfunctions to where it intermittently ignores the IAM conditional for a resource tag. I believe this is the cause of our step function roll back bug.